flask_openapi3/_http.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# @Author  : llc
+# @Time    : 2021/6/21 15:54
+from enum import Enum
+from http import HTTPStatus
+
+HTTP_STATUS = {str(status.value): status.phrase for status in HTTPStatus}
```

Comment:
Why do that though 🤔 you can just get rid of this entirely. It's not used anywhere despite this being labelled an internal module.
Not a blocker just curious.

Reply:
I just want to be consistent with the built-in modules, `HTTPStatus` is not useless, it is used in `utils.py`